crates/ruff_python_parser/src/lexer.rs

```diff
@@ -1585,12 +1591,13 @@ fn is_unicode_identifier_start(c: char) -> bool {
 
 // Checks if the character c is a valid continuation character as described
 // in https://docs.python.org/3/reference/lexical_analysis.html#identifiers
-fn is_identifier_continuation(c: char) -> bool {
+fn is_identifier_continuation(c: char, ascii_only_identifier: &mut bool) -> bool {
```

Comment:
nit: sorry for name recommendations but maybe `is_identifier_ascii_only`?

Reply:
I went for `identifier_is_ascii_only` in https://github.com/astral-sh/ruff/pull/10412/commits/d785be50606df110d848606165e2f7a9aa406835